shared/fs/banner/index.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../../constants/types/fs'
+import {Box, Text, Icon, Button} from '../../common-adapters'
+import {fileUIName, isMobile} from '../../constants/platform'
+import {globalStyles, globalMargins, globalColors} from '../../styles'
+
+type Props = {
+  kbfsEnabled: boolean,
+  inProgress: boolean,
+  showBanner: boolean,
+  path?: Types.Path,
+  onDismiss?: () => void,
+  onInstall: () => void,
+  onUninstall: () => void,
+  openInFileUI?: () => void,
+}
+
+const Banner = isMobile
+  ? () => null
+  : ({kbfsEnabled, showBanner, onInstall, onUninstall, onDismiss, openInFileUI}: Props) => {
+      if (kbfsEnabled && !showBanner) {
+        return null
+      }
+      const iconType = kbfsEnabled ? 'icon-fancy-finder-enabled-132-96' : 'icon-fancy-finder-132-96'
+      const bannerStyle = {
+        ...globalStyles.flexBoxRow,
+        backgroundColor: kbfsEnabled ? globalColors.green : globalColors.blue,
+        height: 176,
+        alignItems: 'center',
+        position: 'relative',
+      }
+      let bannerContent
+      if (kbfsEnabled) {
+        bannerContent = (
+          <Box style={globalStyles.flexBoxColumn}>
+            <Text type="Header" style={textStyle}>
+              {onDismiss && 'Yay! '}Keybase is {onDismiss && 'now '}enabled in your {fileUIName}.
```

Comment:
It seems `onDismiss` is always present. When would it be falsey? Is this for when we have settings that persists the "dismissed" state?

Reply:
It's false in `shared/settings/files/index.js`